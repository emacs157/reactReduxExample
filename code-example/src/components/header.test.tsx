import React from 'react';
import { render, screen } from '@testing-library/react';
import HeaderComponent from './header';

const thisComponent = <HeaderComponent></HeaderComponent>;

it('matches snapshot', () => {
	const { asFragment } = render(thisComponent);
	expect(asFragment()).toMatchSnapshot();
});

it('contains tiltle', () => {
	render(thisComponent);
	const x = screen.getByText(/都道府県/i);
	expect(x).toBeInTheDocument();
});
