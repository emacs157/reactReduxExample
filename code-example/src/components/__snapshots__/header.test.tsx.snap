// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <header
    class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-hip9hq-MuiPaper-root-MuiAppBar-root"
  >
    <div
      class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-dense css-10ltm04-MuiToolbar-root"
    >
      <div
        class="MuiTypography-root MuiTypography-h6 css-1qp5wg9-MuiTypography-root"
      >
        都道府県
      </div>
    </div>
  </header>
</DocumentFragment>
`;
